integration_tests/tests/change_file_content.rs

```diff
@@ -57,9 +56,52 @@ fn test_change_file_content() {
     assert_matches!(change_file_content(), Ok(_));
 }
 
-fn change_file_content_file_not_found() -> Result<(), TestError> {
+fn change_file_content_case_insensitive_username() -> Result<(), TestError> {
```

Comment:
You seem to have added the same handful of tests to a handful of flows, I think it would be more meaningful and less noisy to have a set of well defined tests around new accounts, and then assume (rightfully) that usernames can't change.

Reply:
We discussed this, it turns out Postgres will error if you even just check if a strange string exists in a table, so such a case would result in an internal error. I preferred it this way; what do you think?